timesketch/frontend-ng/src/store.js

```diff
@@ -225,6 +232,44 @@ export default new Vuex.Store({
         })
         .catch((e) => {})
     },
+    updateTimeFilters(context, sketchId) {
```

Comment:
When having a fresh sketch with a new timeline I get this error while testing the PR:
```
SyntaxError: "undefined" is not valid JSON
    at JSON.parse (<anonymous>)
    at parseNode (store.js:246:1)
    at eval (store.js:256:1)
Promise.catch		
updateTimeFilters	@	store.js:270
```

To reproduce: 
1. Create new sketch
2. Upload a timeline
3. Once the timeline is finished, click the search bar and wait for the omni-box to appear.
4. See error on the console.